### PR TITLE
Add maven ingestors

### PR DESCRIPTION
**Pull request by mellisio**

### CODE REVIEW

File: ingestors/maven/maven_central.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package maven
+
+import (
+	"github.com/librariesio/depper/data"
+	log "github.com/sirupsen/logrus"
+	"time"
+)
+
+const mavenCentralSchedule = "1 1 * * MON,THU"
```
Comment (jsonperl) [inline ingestors/maven/maven_central.go:9]: Maybe let's just do these twice a day or something? And extra few runs can't hurt, but minimizing us missing a run.

File: ingestors/maven/maven_parser.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+	response, err := http.Get(parser.URL)
+	if err != nil {
+		return results, err
+	}
+	defer response.Body.Close()
+
+	body, _ := ioutil.ReadAll(response.Body)
+	var mavens []mavenUpdate
+	json.Unmarshal(body, &mavens)
```
Comment (jsonperl) [inline ingestors/maven/maven_parser.go:38]: Ultimately I think this is a cleaner way to marshall json. May as well stick with standard lib eh.

File: ingestors/maven/maven_central.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package maven
```
Comment (jsonperl) [inline ingestors/maven/maven_central.go:1]: I think I'd just leave this all in the ingestors package. No need to get all deeply nested with it I think?

Comment (mellisio) [reply ingestors/maven/maven_central.go:1]: We'll be adding 5 or 6 more at least. That's why I went for it. I can move it back though.

File: ingestors/maven/maven_parser.go (hunk 2 above)
Comment (tiegz) [reply ingestors/maven/maven_parser.go:40]: Agreed, I haven't seen anything yet that couldn't easily get parsed with pure Go yet. (I still wish Go had a native way to parse timestamps in JSON tho!)

File: ingestors/maven/maven_central.go (hunk 3 above)
Comment (jsonperl) [reply ingestors/maven/maven_central.go:1]: I think as long as they're all mave_yada_yada it's easy enough to find em? Just felt really odd to have the `maven.NewMavenCentral()` below?

Comment (tiegz) [reply ingestors/maven/maven_central.go:1]: I'm not strongly opinionated, but Effective Go [says](https://golang.org/doc/effective_go.html#package-names): `Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64. `

**Review by tiegz — APPROVED** [review]

File: ingestors/maven/maven_central.go (hunk 3 above)
Comment (jsonperl) [reply ingestors/maven/maven_central.go]: Yea true, it's just gonna get messy in there. I don't really know what the right answer is on the maven stuffs. Perhaps what @mellisio did originally is more sensible 🤷

**Pull request merged**